plugins/woocommerce/client/legacy/js/frontend/cart.js

```diff
@@ -248,7 +248,10 @@ jQuery( function ( $ ) {
 				data: data,
 				dataType: 'html',
 				success: function ( response ) {
+					var target_id = $( event.currentTarget ).attr( 'id' );
+
 					update_cart_totals_div( response );
+					$( '.cart_totals' ).find( '#' + target_id ).focus();
```

Comment:
The `.cart_totals` element is completely replaced by the HTML that comes with the response. Therefore, we need to select the newly selected radio input and focus it via JS.